package.json

```
@@ -5,7 +5,7 @@
   "main": "lib/main.js",
   "license": "MIT",
   "scripts": {
-    "build": "export NODE_ENV=production && obsidian-plugin build src/main.ts -e esbuild.config.js",
+    "build": "export NODE_ENV=production && obsidian-plugin --with-stylesheet build src/styles.css src/main.ts -e esbuild.config.js",
```

Whoops, screwed up that last commit, I did.

```suggestion
    "build": "export NODE_ENV=production && obsidian-plugin --with-stylesheet src/styles.css build src/main.ts -e esbuild.config.js",
```